docs/docs/04-standard-library/01-cloud/domain.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Domain
+id: domain
+description: A built-in resource for representing a domain in the cloud.
+keywords:
+  [
+    Wing reference,
+    Wing language,
+    language,
+    Wing standard library,
+    Wing programming language,
+    Domain,
+  ]
+sidebar_position: 1
+---
+
+The `cloud.Domain` resource represents the domain that will be configured when creating a website in the cloud.
+
+## Usage
+
+### Defining a domain
+
+```js
+bring cloud;
+
+let domain = new cloud.Domain(
+  domain: "www.example.com",
+);
+
+new cloud.Website(path: "./site", domain: domain);
+```
+
+## Target-specific details
+
+### Simulator (`sim`)
+
+Under the hood, the simulator stores the domain value in memory.
```

Comment:
@eladcon feels like sim.Domain might be where we want to inject the port forwarding behavior into the sdk.